portal/templates/staff_profile_create.html

```diff
@@ -197,7 +197,7 @@ <h4 class="tnth-headline left-indent-top">{{ _("New Staff") }}</h4>
                 }).get();
 
                 _accountArray['organizations'] = orgIDs;
-                _accountArray['roles'] =  [{'name': 'staff'}];
+                _accountArray['roles'] =  [{'name': 'staff'}, {'name': 'write_only'}];
```

Comment:
Might be a good idea to use the strings from the enumerated roles if you can, eg `ROLE.WRITE_ONLY`. Not a big deal but would make refactoring much easier if we decided to rename them...